src/ceph-volume/ceph_volume/terminal.py

```diff
@@ -80,11 +81,33 @@ def make(cls, string):
 class _Write(object):
 
     def __init__(self, _writer=None, prefix='', suffix='', flush=False):
-        self._writer = _writer or sys.stdout
+        if _writer is None:
+            _writer = sys.stdout
+        self._writer = _Write._unicode_output_stream(_writer)
+        if _writer is sys.stdout:
+            sys.stdout = self._writer
```

Comment:
@alfredodeza i realized that i should restore the `sys.stdout` if i am moving away its buffer from it, which practically renders `sys.stdout` closed.